PennMobile/Weekly Schedule/WeeklyScheduleModelManager.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ScheduleNetworkManager.swift
+//  PennMobile
+//
+//  Created by CHOI Jongmin on 16/2/2020.
+//  Copyright © 2020 PennLabs. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+final class WeeklyScheduleModelManager {
+        
+    static let instance = WeeklyScheduleModelManager()
+    private init() {}
+    
+    func fetchModel(_ completion: @escaping (_ model: WeeklyScheduleTableViewModel?) -> Void) {
+        var model: WeeklyScheduleTableViewModel? = WeeklyScheduleTableViewModel()
+        
+        if let courses = UserDefaults.standard.getCourses() {
+            if courses.isEmpty {
+                
+            }
```

Comment:
Let's delete this because it doesn't do anything. We need to handle the state where the user is logged in but we haven't fetched their courses yet, but let's make a TODO and do that in a new PR.